src/App.js

```diff
@@ -1,25 +1,68 @@
 import React from 'react';
+import { Clock } from './Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString(),
+    isClockVisible: true,
+    clockName: 0,
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      if (!this.state.isClockVisible) {
+        return;
+      }
```

Comment:
no need for this check

Reply:
also clearInterval in ComponentWillUnmount
https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class